googlemaps/directions.py

```diff
@@ -24,7 +24,8 @@ def directions(client, origin, destination,
                mode=None, waypoints=None, alternatives=False, avoid=None,
                language=None, units=None, region=None, departure_time=None,
                arrival_time=None, optimize_waypoints=False, transit_mode=None,
-               transit_routing_preference=None, traffic_model=None):
+               transit_routing_preference=None, traffic_model=None,
+               avoid_stopovers=None):
```

Comment:
Should the default be False?

Reply:
Yes, has more sense (doesn't affect the logic)